Tests/WitsmlExplorer.Api.Tests/Services/CredentialsServiceTests.cs

```diff
@@ -168,6 +169,41 @@ public void GetUsernames_ValidTokenUserValidBasicUserValidRole_ReturnUsernames()
             Assert.Equal((upn, "basicuser"), (tokenUser, basicUser));
         }
 
+        [Fact]
+        public void GetCredentialsCookieFirst_ValidCookie_ReturnCookieCreds()
+        {
+            string host = "http://some.targeturl.com/";
+            ServerCredentials scTarget = new() { UserId = "cookieuser", Password = "cookiepass", Host = new Uri(host) };
+            IEssentialHeaders essentialHeaders = CreateEssentialHeaders(scTarget, null);
+            ServerCredentials serverCreds = _credentialsService.GetCredentialsCookieFirst(essentialHeaders, EssentialHeaders.WitsmlTargetServer).Result;
+
+            Assert.True(serverCreds.UserId == "cookieuser" && serverCreds.Password == "cookiepass" && serverCreds.Host.ToString() == host);
```

Comment:
Shouldn't each comparison be its own assert?

Reply:
It might ? I think that is up for debate. I was here following the "only one assert pr test" flavour. 
This is something I can split up if it improves readability

Reply:
I see your point. I am more used to something like this https://softwareengineering.stackexchange.com/a/171502
I guess if the contents of the assert are abstracted as "boolean receivedValidCookie" then a single assert should hold.

Reply:
I have no strong feelings for either, so I will split and we can follow that going forward 👍